src/data.js

```diff
@@ -4149,6 +4149,32 @@ module.exports = [
       'Azure',
     ],
   },
+  {
+    name: 'Rajanand Ilangovan',
+    description:
+      'Business intelligence developer with over a decade of experience in designing and developing databases, ETL and reporting solutions. I am also a Microsoft certified trainer. 👉 https://rajanand.org/bio',
+    url: 'https://rajanand.org/uses',
+    twitter: '@iRajanand',
+    emoji: '🎯',
+    country: '🇮🇳',
+    computer: 'windows',
+    phone: 'android',
+    tags: [
+      'Business Intelligence',
+      'Cloud',
+      'Data',
+      'SQL'
+      'SQL Server',
```

Comment:
@rajanand it looks like you're missing a comma here. Will you take a look and fix it?